src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -7282,11 +7282,17 @@ void Player_StopCutscene(Player* this) {
     }
 }
 
-s32 func_808387A0(PlayState* play, Player* this) {
+/**
+ * If appropriate, setup action for performing a `csAction`
```

Comment:
If using doxygen style, should this have an `@brief` and `@params` since you are using `@return`?

Reply:
Sure, I added `@brief` but I don't think `@params` is needed when the args are just `play` and `this` like nearly every other function